### DISCUSSION TITLE
Is there a way to specify a directory other than dist when running nuxi generate?

### DISCUSSION THREAD

**Discussion by mcfarljw** [nuxt/nuxt#18465] [Questions] [answered]
I remember https://nuxtjs.org/docs/configuration-glossary/configuration-generate#dir was a thing with Nuxt 2, but I'm not finding a configuration for it in the Nuxt 3 documentation. I'm looking to change the `dist` directory to something like `dist/flavor1` and `dist/flavor2`.

**Comment by TheAlexLichter** [comment, marked as answer]
No, right now this is hardcoded in Nuxt https://github.com/nuxt/nuxt/blob/3c3291226dde769b0e25c8dd3e7049f2b5934a87/packages/nuxt/src/core/nitro.ts#L150 and Nitro https://github.com/unjs/nitro/blob/e7b250f2591120e75a152565d9f7d5badb5ef99e/src/rollup/config.ts#L60.

What you *can* do though is hooking up into the `build:done` hook and then move files around "manually".

If you are interested in an implementation of changing the `distDir` via config, feel free to raise an issue ☺️
